pkg/cmd/agent-task/list/list.go

```diff
@@ -0,0 +1,150 @@
+package list
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/cli/cli/v2/internal/gh"
+	"github.com/cli/cli/v2/internal/tableprinter"
+	"github.com/cli/cli/v2/pkg/cmd/agent-task/capi"
+	"github.com/cli/cli/v2/pkg/cmd/pr/shared"
```

Comment:
How about `prShared` as a little remark that this comes from another command set?

```suggestion
	prShared "github.com/cli/cli/v2/pkg/cmd/pr/shared"
```

If you decided to apply this then you should also change L107 as well.

Reply:
Done in 0de5cf24f070fd6849712a95a8267e8ce00dc94a